src/home/NotificationCenter.test.tsx

```diff
@@ -31,6 +32,10 @@ jest.mock('src/navigator/NavigationService', () => ({
   navigate: jest.fn(),
 }))
 
+jest.useFakeTimers()
```

Comment:
nit: fake timers are enabled by default in jest_config.ts so no need to declare it here